astropy/modeling/tests/test_functional_models.py

```diff
@@ -148,3 +149,13 @@ def test_Scale_inverse():
 def test_Shift_inverse():
     m = models.Shift(1.2345)
     assert_allclose(m.inverse(m(6.789)), 6.789)
+
+
+def test_Voigt1D():
```

Comment:
Ah, this also needs to be prepended with:
```python
@pytest.mark.skipif("not HAS_SCIPY")
```
making sure to also add `from ...tests.helper import pytest` to the imports in the module.